.github/workflows/kit-bootstrap.yml

```diff
@@ -0,0 +1,53 @@
+name: This workflow automates the build, testing, and bootstrapping of a test environment on Kubernetes using KIT CLI
+on:
+  workflow_run:
+    workflows: [trigger-kit-CI]
+    types: [completed]
+permissions:
+  id-token: write
+  pull-requests: write
+  contents: read
+jobs:
+  kit:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/install-go-and-dependencies
+      - uses: aws-actions/configure-aws-credentials@v1
```

Comment:
I'd expect you to reference things like `AWS_ACCESS_KEY_ID_PERFLAB`. Why are you not using them now?

Reply:
while AWS access keys can be a valid way to authenticate and authorize access to resources, OIDC provides a more secure, scalable, and manageable solution.